openedx/core/djangoapps/credit/tests/test_api.py

```diff
@@ -73,6 +84,7 @@ def add_credit_course(self, enabled=True):
         return credit_course
 
 
+@skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Test only valid in LMS')
 @ddt.ddt
 class CreditRequirementApiTests(CreditApiTestBase):
```

Comment:
Please add at least one test case that verifies that the email gets sent.  Django has some nice tools for unit testing email: https://docs.djangoproject.com/en/1.8/topics/testing/tools/#email-services